src/bundle/ui-dev/src/modules/universal-discovery/hooks/useSearchByQueryFetch.js

```diff
@@ -41,7 +40,6 @@ export const useSearchByQueryFetch = () => {
             contentNameCriterion = null,
         ) => {
             const handleFetch = (response) => {
-                setMarkedLocationId(null);
```

Comment:
Note: we do not want to reset the marked location id with every search, so I moved it to a more specific place.